presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -80,6 +80,9 @@ CHAR
 ----
 
     Fixed length character data. Char type without length specified has default length of 1.
```

Comment:
What's the convention around using all caps?  Should the word "char" here be "CHAR" instead?